concepts/objects/about.md

```diff
@@ -0,0 +1,266 @@
+# About
+
+## Explanation
+
+Besides primitive data types like `number` and `string`, there is another important data type in JavaScript called `object`.
+Objects are collections of key-value pairs.
+As such, they can be used as what is referred to as maps or dictionaries in other languages.
```

Comment:
```suggestion
As such, they can be used as what is often referred to as maps or dictionaries in other languages.
```

structs in other languages are also sometimes used purely as a collection. It's rare but imo making this statement slightly weaker clears up that it doesn't _always_ translate to a map or dictionary even if in many languages it does.

The difference to classes with methods in the context of this exercise is clearer to me now, though, so I think it'd be fine as it is too.

Reply:
fixed in all three places